run_dir/static/js/projects_components.js

```diff
@@ -0,0 +1,854 @@
+export const vProjectDataField = {
+    name: 'v-project-data-field-tooltip',
+    // A single dt field with a dd field
+    props: ['field_name', 'project_data'],
+    computed: {
+        field_source() {
+            if (this.project_data['field_sources'] == undefined) {
+                return ''
+            }
+            return this.project_data['field_sources'][this.field_name]
+        },
+        data_value() {
+            if (this.field_name in this.project_data) {
+                return this.project_data[this.field_name]
+            } else {
+                return '-' // Missing value
+            }
+        }
+    },
+    mounted: function() {
+        // Init bootstrap tooltip
+        this.$nextTick(function () {
+            this.tooltip = new bootstrap.Tooltip(this.$el);
+        })
+    },
+    template:
+    /*html*/`
+        <span data-toggle="tooltip" data-html="true" data-placement="right" :title="field_source">{{data_value}}</span>
+    `
+}
+
+export const vProjectDetails = {
+    name: 'v-project-details',
+    // The single_project_mode is to be used when only a single project is used for the app
+    // and the project cards are not shown
+    props: ['project_id', 'as_modal', 'single_project_mode', 'user'],
+    data: function() {
+        return {
+            active_tab: 'project-running-notes-pane'
+        }
+    },
+    computed: {
+        project_data() {
+            return this.$root.project_details[this.project_id]
+        },
+        project_data_exists() {
+            return this.project_data != undefined
+        },
+        running_notes() {
+            if (this.project_id in this.$root.running_notes) {
+                return this.$root.running_notes[this.project_id]
+            }
+        },
+        project_samples() {
+            return this.$root.project_samples[this.project_id]
+        },
+        project_comment() {
+            if (this.project_data['project_comment'] == undefined) {
+                return ''
+            }
+            return marked.parse(this.project_data['project_comment'])
+        },
+        status_bg_class() {
+            if (this.project_data['status'] == 'Aborted') {
+                return 'bg-danger'
+            } else if (this.project_data['status'] == 'Closed') {
+                return 'bg-success'
+            } else if (this.project_data['status'] == 'Ongoing') {
+                return 'bg-info'
+            } else if (this.project_data['status'] == 'Reception Control') {
+                return 'bg-warning'
+            } else if (this.project_data['status'] == 'Pending') {
+                return 'bg-light text-dark'
```

Comment:
I feel bg-light is a bit too light, its hardly visible

Reply:
Fixed it!